huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_instance_test.go

```diff
@@ -230,3 +299,61 @@ resource "huaweicloud_ddm_instance" "test" {
 }
 `, testDdmInstance_base(name), updateName)
 }
+
+func testDdmInstance_prepaid_basic(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_ddm_instance" "test" {
+  name              = "%[2]s"
+  flavor_id         = data.huaweicloud_ddm_flavors.test.flavors[0].id
+  node_num          = 2
+  engine_id         = data.huaweicloud_ddm_engines.test.engines[0].id
+  vpc_id            = huaweicloud_vpc.test.id
+  subnet_id         = huaweicloud_vpc_subnet.test.id
+  security_group_id = huaweicloud_networking_secgroup.test.id
+  admin_user        = "test_user_1"
+  admin_password    = "test_password_123"
+
+  charging_mode = "prePaid"
+  period_unit   = "month"
+  period        = 1
+  auto_renew    = false
```

Comment:
please update your code `"is_auto_renew": utils.ValueIngoreEmpty(d.Get("auto_renew")),` (L328), we don't know whether the default value is true or false.

Reply:
done